### PR TITLE
Android isAvailable method checks Custom Tab support

### PR DESCRIPTION
**Pull request by aitorct**
Android's isAvailable method now checks if any installed browser supports Custom Tabs.

This solves issue https://github.com/proyecto26/react-native-inappbrowser/issues/91.

### CODE REVIEW

**Comment by wong2** [comment]
hi, any updates?

**Comment by jdnichollsc** [comment]
Let me know if it Works for AndroidX 🙂

**Comment by ssuchanowski** [comment]
any plan for merging this?

**Comment by jdnichollsc** [comment]
@ssuchanowski do you know if it works for AndroidX too?

**Comment by ssuchanowski** [comment]
Hadn't tested it but it seems like this service name is mentioned here https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsService which is an android x documentation.

**Comment by jdnichollsc** [comment]
Ok, let me check this weekend, thanks to confirm!

**Comment by ssuchanowski** [comment]
hey @jdnichollsc did you have time to take a look?

There is also another PR (https://github.com/proyecto26/react-native-inappbrowser/pull/160) with a simple fix.

**Comment by jdnichollsc** [comment]
@ssuchanowski do you know if we need to set the package? Example: https://stackoverflow.com/a/33041870/1532821

**Comment by ssuchanowski** [comment]
No idea

File: android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java [hunk 1]
```diff
@@ -162,6 +164,12 @@ public void close() {
         ChromeTabsManagerActivity.createDismissIntent(currentActivity));
   }
 
+  public void isAvailable(Context context, final Promise promise) {
+    Intent serviceIntent = new Intent("android.support.customtabs.action.CustomTabsService");
```
[the remark this answers is not in the document]
Comment (aitorct) [reply android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:168]: Hey! I've been too busy lately, sorry for the late reply.
Yes, it works for Android X. There was a bug were actually jetifier was wrongly replacing this package, but looks like it's now fixed, so it should work just fine. https://issuetracker.google.com/issues/119183822

**Comment by aitorct** [comment]
Just double-checked, it's working properly! Fixed the merge conflicts as well.

**Pull request merged**